### ISSUE TITLE
Error when trying to run docker

### ISSUE THREAD

**Issue by ghost** [meteorhacks/meteord#82]
Hi there,

always when I try to run docker, no matter what, it returns an error:

*/Users/username/.nvm/versions/node/v5.5.0/lib/node_modules/docker/src/docker.js:259
        throw err;
        ^

Error: ENOENT: no such file or directory, lstat '/Users/username/run'
    at Error (native)*

I've tried your tutorials and also the ones from the docker.com page, nothing seems to help.

I am running on a Macbook Pro, docker-machine is running.

What can I do here?

**Comment by jupiterkenji** [comment]
What's your docker run command?

**Comment by ghost** [comment]
Hi @stewiegaribaldi 

I was pretty much going by the tutorial provided in _https://github.com/meteorhacks/meteord_

```
 docker build -t myname/app .
```

It returns the error:
_Error: Error calling `pygmentize`: Error: no lexer for alias 'typescript' found_

**Comment by jupiterkenji** [comment]
Sorry, I'm not sure, I did the 2.1 From a Local Mount and it works fine.

**Comment by ghost** [comment]
just a second, i will try 2.1

**Comment by ghost** [comment]
docker actually looks for a file called run instead of running the container. that is the error msg:
_Error: ENOENT: no such file or directory, lstat '/Users/tseckert/My_Proj_Dir/run'
    at Error (native)_

**Comment by jupiterkenji** [comment]
I have a feeling it is your code where you try to read that file. May be try to put the run file there and see what it says?

**Comment by ghost** [comment]
Yes, but is run in _docker run_ a file or a command?
Because the error states that the file run is not there but usually it
should be a command is it?

Am Donnerstag, 17. März 2016 schrieb stewiegaribaldi :

**Comment by jupiterkenji** [comment]
I think you need to do:
docker run -v /your_computer_folder/run:/Users/tseckert/My_Proj_Dir/run ...

**Comment by ghost** [comment]
Also I don't know whether it is because my app is a mobile Cordova app? Maybe I should try a small testing project first and see if that works.

**Comment by jupiterkenji** [comment]
I agree, go to basic when things don't work. but I think it's because you don't attach your docker volume -v as I mentioned previously. Let me know how it goes
